From 2e8b78f19ff14041eae0742fb0e7510cd96fccc4 Mon Sep 17 00:00:00 2001
From: Michael Stack <stack@apache.org>
Date: Sat, 8 Dec 2012 21:41:02 +0000
Subject: [PATCH 057/202] HBASE-7301 Force ipv4 for unit tests

Reason: Improvement
Author: Jean-Daniel Cryans
Ref: CDH-9406

git-svn-id: https://svn.apache.org/repos/asf/hbase/branches/0.94@1418758 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 3f58c188e3d424cfe407a812ea68d15e526e4e7f)
---
 pom.xml |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/pom.xml b/pom.xml
index c6bbdf6..1a4cfa6 100644
--- a/pom.xml
+++ b/pom.xml
@@ -396,7 +396,8 @@
 
           <configuration>
             <forkedProcessTimeoutInSeconds>900</forkedProcessTimeoutInSeconds>
-            <argLine>-enableassertions -Xmx1900m -Djava.security.egd=file:/dev/./urandom</argLine>
+            <argLine>-enableassertions -Xmx1900m
+              -Djava.security.egd=file:/dev/./urandom -Djava.net.preferIPv4Stack=true</argLine>
             <redirectTestOutputToFile>${test.output.tofile}</redirectTestOutputToFile>
           </configuration>
         </plugin>
-- 
1.7.0.4

